blog.html

```diff
@@ -0,0 +1,70 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!--#include virtual="includes/_header.htm" -->
+<body class="page-project ">
+    <!--#include virtual="includes/_top.htm" -->
+    <div class="content">
+        <!--#include virtual="includes/_nav.htm" -->
+        <div class="right">
+            <h1 class="content-title">Blog</h1>
```

Comment:
should we call this "news" or "announcements" instead of "blog"? Are we planning to post non-release related content on this page as well?

Reply:
@mimaison thoughts about this?

Reply:
No plans currently but if anybody is interested in posting non-release content, why not!

Reply:
ok, in that case let's keep it as a blog.